### PR TITLE
feat: vocalize when item is added to wish list (#15216)

### LINKED ISSUE #15216: Add product to wish list lacks vocalized confirmation when product is added
**Describe the bug**
When the user adds a product to the wish list, there's no audible confirmation of this happening.

**Tell us the version of Spartacus**
- Library version: develop

Screen Shot 2022-02-22 at 10 35 53 AM

**Expected behaviour**
- Vocalized confirmation of item being added to wish list: _"This item has been added to your wish list"._

### PR DESCRIPTION
**Pull request by BenjaminEmiliani**
Closes #15216

### CODE REVIEW

**Review by dydome — DISMISSED** [review]
That is pretty strange but it is not working for me. @BenjaminEmiliani please ping me we can test it together.

**Review by developpeurweb — DISMISSED** [review]
Works well for JAWS/NVDA/VO

**Review by pawelfras — DISMISSED** [review]

File: projects/storefrontlib/shared/components/assistive-technology-message/assistive-technology-message.directive.ts [hunk 1]
```diff
@@ -40,10 +45,25 @@ export class AtMessageDirective {
...
+      this.globalMessageService
+        .get()
+        .pipe(take(1))
+        .subscribe((globalMessageEntities: GlobalMessageEntities) => {
+          const current =
+            globalMessageEntities &&
+            globalMessageEntities[GlobalMessageType.MSG_TYPE_ASSISTIVE];
+
+          // Override current assitive message.
+          if (current) {
```
Comment (Pucek9) [inline projects/storefrontlib/shared/components/assistive-technology-message/assistive-technology-message.directive.ts]:
```suggestion
          // Override current assitive message.
          if (globalMessageEntities?[GlobalMessageType.MSG_TYPE_ASSISTIVE) {
```

**Review by Pucek9 — APPROVED** [review]

**Review by Pucek9 — COMMENTED** [review]
you might consider removing timeout from global message config (MSG_TYPE_ASSISTIVE)

**Comment by Pucek9** [comment]
> you might consider removing timeout from global message config (MSG_TYPE_ASSISTIVE)

ok, confirmed that then it will read it every time there are messages of a different type

**Pull request merged**